backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java

```diff
@@ -144,6 +172,24 @@ public Endpoint createEndpoint(@Context SecurityContext sec, @NotNull @Valid End
             throw new BadRequestException("Properties is required");
         }
 
+        if (obEnabled) {
+            // TODO NOTIF-429 - see similar in EndpointResources#createEndpoint
+            String endpointSubType = endpoint.getSubType() != null ? endpoint.getSubType() : endpoint.getProperties(CamelProperties.class).getSubType();
+
+            if (endpointSubType.equals("slack")) {
+                CamelProperties properties = endpoint.getProperties(CamelProperties.class);
+                String processorName = "p-" + endpoint.getAccountId() + "-" + UUID.randomUUID();
+                properties.getExtras().put(OB_PROCESSOR_NAME, processorName);
+                String processorId = setupOpenBridgeProcessor(endpoint, properties, processorName);
+
+                // TODO find a better place for these, that should not be
+                //       visible to users / OB actions
+                //       See also CamelTypeProcessor#callOpenBridge
+                properties.getExtras().put(OB_PROCESSOR_ID, processorId);
```

Comment:
Did you check if Hibernate ORM saves the properties changes into the DB? It is supposed to work like that since we're talking about a persistent entity within a JTA transaction, but since we recently moved to Hibernate ORM, maybe it's worth checking.